Add latest version in npmignore (#6)
wanadev/hermes-worker: README.md

    npm install hermes-worker

> Please not currently version not works on Node (Blob is not available in node)

Reviewer: ```suggestion
Please note that current version does not work with NodeJS (Blob not available)
```
